### ISSUE TITLE
Plugin could not be activated because it triggered a fatal error.

### ISSUE THREAD

**Issue by Huyeng** [OpenDevelopmentMekong/wp-odm_solr#80]
Get error once try to activate the wp-odm_solr plugin in local:
Fatal error: Class 'Solarium\Client' not found in \wp-content\plugins\wp-odm_solr\utils\solr-wp-manager.php on line 48

**Comment by acorbi** [comment]
> Installed composer

Have you installed composer or run ```composer install``` to install dependencies? Have you checked that you have the **vendor** folder under  \wp-content\plugins\wp-odm_solr\ ? 

You can try runnning ```composer update``` on the folder.

**Comment by Huyeng** [comment]
Yes. I did. After running the the composer install, there are some directories and files were created. And I also tried to run the composer update as well, but the plugin can't still not activated. See the screenshot. 

![screenshot_3](https://user-images.githubusercontent.com/6613530/26858882-46430b6a-4b5e-11e7-92de-9c11f26a87be.jpg)

![screenshot_4](https://user-images.githubusercontent.com/6613530/26858973-d461bafe-4b5e-11e7-864e-dafe9a97f5bb.jpg)

**Comment by acorbi** [comment]
@Huyeng if you solved this issue. Could you please provide details for the solution here and close the issue so it gets documented?

**Comment by Huyeng** [comment]
Nope, the issue was not fixed yet. I can't find what cause the problem now. I will do when I have some free time. Currently, it was pending while I'm working on other tasks.
